src/wallet/rpc/coins.cpp

```diff
@@ -672,7 +672,7 @@ RPCHelpMan listunspent()
             std::unique_ptr<SigningProvider> provider = pwallet->GetSolvingProvider(scriptPubKey);
             if (provider) {
                 if (scriptPubKey.IsPayToScriptHash()) {
-                    const CScriptID& hash = CScriptID(std::get<ScriptHash>(address));
+                    const CScriptID& hash = ToScriptID(std::get<ScriptHash>(address));
```

Comment:
Seems strange for this to be a reference (and rely on lifetime extension)?

Reply:
Dropped the reference